src/utils/hooks/useCategories.js

```diff
@@ -0,0 +1,45 @@
+import { useState, useEffect } from "react";
```

Comment:
I don't have much to say about the hooks that fetch the API, as I believe the API itself is more complicated to use than it should, there's been multiple people with troubles fetching filtered information from one or more categories because is not that clear on the use of it. And I see you and almost everyone is just reusing the hook that the project had.

Reply:
I also thought the api hook was a little weird, I was planning on trying to rework it with axios and handle the categories with the params. Reading prismic docs I found out that you can apply a query on top of another query and that's what I did haha